libraries/matrix/src/main/kotlin/io/element/android/libraries/matrix/di/MatrixModule.kt

```diff
@@ -33,4 +38,14 @@ object MatrixModule {
     fun providesRustAuthenticationService(baseDirectory: File): AuthenticationService {
         return AuthenticationService(baseDirectory.absolutePath)
     }
+
+    @Provides
+    @SingleIn(AppScope::class)
+    fun provideMatrixDatabase(@ApplicationContext context: Context): Database {
+        val name = "matrix_database"
```

Comment:
Maybe `credentials_database` instead? What are we planning to store there in the future?

Reply:
Yeah I guess we could have an other name. Also the `Database` class should have a better name?

Reply:
Yes, I will change it to something better.